src/components/novoproblems/ProblemView.svelte

```diff
@@ -370,14 +370,23 @@
                       {problem}
                     />
                     {#each problem.FullChildren as [_, child]}
-                      {#if childProblemFilter}{#if child.FullTextSearch(childProblemFilter) > 0.65}
+                      {#if child.Status != "closed"}
```

Comment:
We need to search children that are also closed, the point is to show the user any relevant problem that has been logged in the past so that they can see it for reference (even if it is closed)